### PR TITLE
Add SDBlockDevice example

### PR DESCRIPTION
**Pull request by juhaylinen**

### CODE REVIEW

File: SDBlockDevice/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# SDBlockDevice example
+
+SDBlockDevice usage example for Mbed OS
+
```
Comment (melwee01) [inline SDBlockDevice/README.md]: The .cpp appears underneath as an embedded element, right?

File: SDBlockDevice/main.cpp [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+//     MOSI (Master Out Slave In)
+//     MISO (Master In Slave Out)
+//     SCLK (Serial Clock)
+//     CS (Chip Select)
+SDBlockDevice sd(MBED_CONF_SD_SPI_MOSI, MBED_CONF_SD_SPI_MISO, MBED_CONF_SD_SPI_CLK, MBED_CONF_SD_SPI_CS);
+uint8_t block[512] = "Hello World!\n";
+
+int main()
+{
+    // call the SDBlockDevice instance initialisation method.
```
Comment (melwee01) [inline SDBlockDevice/main.cpp]: Would recommend consistently using periods at the end of comments or not, rather than a mixture.

Comment (melwee01) [reply SDBlockDevice/main.cpp]: Same with capitalized first letters of comments.

Comment (juhaylinen) [reply SDBlockDevice/main.cpp]: Fixed

**Pull request merged**